pages/Nix/_index.md

```diff
@@ -7,9 +7,10 @@ To install Hyprland on NixOS, we provide a NixOS and a Home Manager module.
 
 {{< callout title=Note >}}
 
-- _(Required) NixOS Module_: enables critical components needed to run Hyprland
+- _**(Required)** NixOS Module_: enables critical components needed to run Hyprland
   properly
-- _(Optional) Home-manager module_: lets you declaratively configure Hyprland
+- _(Optional) Home-manager module_: lets you configure Hyprland declaratively through home-manager.
+  - _Note that this does_ not _install Hyprland; since it would require system-level changes such as adding a session file for your display manager. This is handled by the NixOS module once you enable it._ 
```

Comment:
I think this is a bit unclear. Hyprland *is* installed, and can be run from the tty. It's only the session file that isn't installed.

Reply:
Good to know, I wasn't aware of this. If you have any suggestions for alternative wording let me know, otherwise I'll brew something up